tests/test_plot.py

```diff
@@ -330,6 +330,17 @@ def test_kde_plot_1d(plot_1d):
             pass
 
 
+def test_ncompress_kde_1d():
+    fig, ax = plt.subplots()
+    np.random.seed(0)
+    data = np.random.randn(5000)
+    line0, = kde_plot_1d(ax, data, q='5sigma', ncompress=5000)
+    line1, = kde_plot_1d(ax, data, q='5sigma', ncompress=1000)
+    x2y = interp1d(line0.get_xdata(), line0.get_ydata())
+    assert_allclose(line1.get_ydata(), x2y(line1.get_xdata()))
+    plt.close("all")
```

Comment:
This tests the effect of `ncompress` and works nicely for the 1d case, but I can't quite find an equivalent test for the 2d case, where we don't plot the full data, but only the iso contour lines...
Do we need an `ncompress` test for the 2d case? (We didn't have one before...)